web/package.json

```diff
@@ -121,8 +124,8 @@
     "react-dom": "^17.0.2",
     "react-i18next": "^11.8.11",
     "react-modal": "^3.14.4",
-    "react-router": "^6.14.0",
-    "react-router-dom": "^6.14.0"
+    "react-router": "^5.3.0",
+    "react-router-dom": "^5.3.0"
```

Comment:
here also this is downgrades

Reply:
(we must make sure it won't introduce security regressions)

Reply:
Yes. As written in the description, `dynamic-plugin-sdk-webpack` is now doing checks on our dependencies and forces compatible versions with OCP Console ones.